packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -171,9 +172,12 @@ function ChatWidget(props: ChatWidgetProps): JSX.Element {
       setUnreadMessagesByTeamForPatient(unreadMessages)
     }
 
-    fetchMessages()
-    if (isUserPatient) {
-      fetchUnreadMessagesCountForPatient()
+    if (dropdownTeamId !== teamIdForWhichMessagesHaveBeenFetched.current) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Maybe `messagesTeamId` would be a shorter and equally understandable name? (or `fetchedMessagesTeamId` if we want to keep the notion of API fetching)

```suggestion
    if (dropdownTeamId !== messagesTeamId.current) {
```